estate/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Real Estate",
+    "version": "1.0",
+    "description": "Real Estate Management System",
+    "summary": "Real Estate Management System",
+    "author": "Lud0do1202",
```

Comment:
```suggestion
    'author': 'Odoo',
```

Usually, we don't add the author on modules.
If you really want to keep it clean, please use "Odoo" (or "Odoo SA", ...) as your work for Odoo.